db/migrate/20200205154839_update_vssc_extract_date_value.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateVsscExtractDateValue < ActiveRecord::Migration[5.2]
+    def up
+      DrivetimeBand.update_all("vssc_extract_date='2019-10-31 00:00:00'")
+    end
+  
+    def down
```

Comment:
This doesn't need a corresponding down because we're relying on dropping the column in the preceding migration to take care of cleaning up the data, right? Is there any danger with that? I can't think of anything pressing—but wanted to mention it just in case.

Reply:
That tracks my understanding. I could have used the `down` to set the value back to NULL before removing the column, but I couldn't find any reason to actually do so.